lib/runCall.js

```diff
@@ -370,7 +384,15 @@ module.exports = function (opts, cb) {
 
   function createStorageData (data) {
     var storageData = new msgPb.StorageData()
-    storageData.setData(new Uint8Array(data))
+    var dataArray = new Uint8Array(data)
+    if (dataArray[0] >= 0x80) {
+      // Must send a positive signed number
+      var c = new Uint8Array(1 + dataArray.length)
+      c[0] = 0
+      c.set(dataArray, 1)
+      dataArray = c
+    }
+    storageData.setData(dataArray)
```

Comment:
Hmmm, this looks weird to me. Why are we storing a `Uint8Array`, EVM uses Uint256 for storage. Or is this for setting a single storage element?